packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -223,11 +214,12 @@ void queryDispatch(
 }
 
 template <typename DeviceType, typename Query>
-void queryDispatch( BoundingVolumeHierarchy<DeviceType> const bvh,
+void queryDispatch( Details::SpatialPredicateTag,
+                    BoundingVolumeHierarchy<DeviceType> const bvh,
                     Kokkos::View<Query *, DeviceType> queries,
                     Kokkos::View<int *, DeviceType> &indices,
                     Kokkos::View<int *, DeviceType> &offset,
-                    Details::SpatialPredicateTag )
+                    int buffer_size = 30 )
```

Comment:
Let me know if you have a better idea for the default value.

Reply:
I think that we should just use zero since we don't know what a reasonable value should be.